src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,36 +1,38 @@
 package edu.kis.vh.nursery;
 
 public class DefaultCountingOutRhymer {
+    public static final int END = 11;
+    public static final int CAPACITY = END+1;
+    public static final int INITIAL_VALUE = -1;
+    public static final int EMPTY = -1;
 
-    public static final int begin = -1;
-    public static final int end = 11;
-    private int[] NUMBERS = new int[12];
+    private final int[] NUMBERS = new int[CAPACITY];
 
-    public int total = begin;
+    public int total = INITIAL_VALUE;
 
     public void countIn(int in) {
         if (!isFull())
             NUMBERS[++total] = in;
     }
 
     public boolean callCheck() {
-        return total == begin;
+        return total == EMPTY;
     }
 
 
     public boolean isFull() {
-        return total == end;
+        return total == END;
     }
 
     protected int peekaboo() {
         if (callCheck())
-            return begin;
+            return EMPTY;
```

Comment:
Do peekaboo i do countOut -1 nie było do empty tylko to po prostu return value, więc sens nazwy zmiennej jest zły, to o to chodziło z tymi dwiema różnymi stałymi do -1

Reply:
Oki, dokończę w domu i odeślę